test/test_jit.py

```diff
@@ -10290,6 +10289,11 @@ def forward(self, x, y):
     'test___getitem___adv_index_sub_2',
     'test___getitem___adv_index_sub_3',
     'test___getitem___adv_index_var',
+
```

Comment:
Uhh, this is a regression in behavior, right? We should not commit until it is resolved.

Reply:
the tracing part has been fixed by my recent commit to emit the List[Optional[Tensor]], so I removed the changes and now it does not break actual users.